Gemfile

```diff
@@ -20,6 +20,7 @@ group :production do
 end
 
 group :development, :test do
+  gem 'byebug'
```

Comment:
Let's pull out `byebug`. Rails 7 now replaces with this `debugger` that acts the same way without an extra gem.

Reply:
oh, cool - sure thing